.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F9E8 Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+<!-- Please fill out as much of the template as you can -->
+
+<!-- Start below this comment. -->
+
+
+**Please describe the desired behavior.**
+Describe the solution you'd like.
```

Comment:
I think the "feature" would be more appropriate here than "solution".
The word "solution" sounds like we are some business agency.